less/project/theme-extras.less

```diff
@@ -192,3 +188,57 @@
 }
 
 }
+
+// --------------------------------------------------
+// Common
+// --------------------------------------------------
+
+// Background color mixin
+// Can be added to course, contentObject, article, block, or component to enable
+// e.g. 'bg-color black'
+// Note: both arguments must be predefined variables
+// --------------------------------------------------
+.bg-color-mixin(black, white);
+.bg-color-mixin(background, background-inverted);
+.bg-color-mixin(transparent-light, black);
+.bg-color-mixin(transparent-dark, white);
+
+.bg-color-mixin(@color, @color-inverted) {
+
+.bg-color.@{color} {
```

Comment:
indentation looks off here

Reply:
A little bit of personal preference creeping in to reduce the overall mixin indentation - am happy to indent if you think we should @eleanor-heath

Reply:
No it's ok to leave it, I would indent it but it is only a very minor and the code is still easy to follow.